### ISSUE TITLE
6.0.0 missing file crypter

### ISSUE THREAD

**Issue by natew** [pubkey/rxdb#293]
Just a heads up, ran an install and data-migrator is having trouble finding crypter:

![image](https://user-images.githubusercontent.com/12100/30609201-93537a52-9d2f-11e7-83a7-e7d1fa4417d3.png)

**Comment by natew** [comment]
Tried installing direct from `github:pubkey/rxdb#dbd9acb236d938ca644f864104e5bc35eb8b3280`:

```
/Users/nw/projects/motion/orbit/node_modules/rxdb/dist/lib/plugins/adapter-check.js:119
    core: core
          ^
ReferenceError: core is not defined
    at Object.<anonymous> (/Users/nw/projects/motion/orbit/node_modules/rxdb/dist/lib/plugins/adapter-check.js:119:11)
```

**Comment by pubkey** [comment]
Hm strange. I tested the rxdb from npm with the ng2 and the vue-example. Everything works there. Does the `crypter.js` exist in your rxdb-folder?

**Comment by pubkey** [comment]
Maybe this is equal to [#296](https://github.com/pubkey/rxdb/issues/296)

**Comment by natew** [comment]
Fixed in 6.0.1
